TicketSystem/TicketSystem/Controllers/MessageController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using TicketSystem.Data;
+using TicketSystem.Models;
+
+namespace TicketSystem.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class MessageController : ControllerBase
+    {
+        private readonly ILogger<MessageController> _logger;
+        private readonly ApplicationContext _context;
+
+        public MessageController(ILogger<MessageController> logger)
+        {
+            _context = new ApplicationContext();
```

Comment:
I suppose it will be better to add all architecture ASAP, I mean, for example, the message controller goes to the message service from BLL library, which goes to the message repository from DAL which calls the context and makes some operations with the storage. Also, our lead, Mr. Antonov has an example in his own GitHub repository (except controllers part) https://github.com/nantonov/3TierArchitectureSolutionTemplate.
Hope you will add the necessary changes in the next PRs, let me know if you need additional explanations or examples.

Reply:
It's all part of defined plan for studying, so don't be worry about that. We'll see fully configured architecture at the end of this plan)